### PR TITLE
Optee master+warp7

### PR DESCRIPTION
**Pull request by bryanodonoghue**

### CODE REVIEW

**Comment by bryanodonoghue** [comment]
@MrVan can you find a few minutes to review this ?

File: core/arch/arm/plat-imx/conf.mk [hunk 1]
```diff
@@ -9,6 +9,7 @@ mx6d-flavorlist =
 mx6dl-flavorlist = mx6dlsabresd
 mx6s-flavorlist =
 mx7-flavorlist = mx7dsabresd
+mx7-flavorlist += mx7swarp7
```
Comment (MrVan) [inline core/arch/arm/plat-imx/conf.mk]: `mx7-flavorlist = mx7dsabresd mx7swrap7` is better.

Comment (bryanodonoghue) [reply core/arch/arm/plat-imx/conf.mk]: yes thanks I'll get to this tomorrow (today look at the time 02:58) :)

Comment (bryanodonoghue) [reply core/arch/arm/plat-imx/conf.mk]: done (completely forgot to update this before now - oops)

**Comment by MrVan** [comment]
`Reviewed-by: Peng Fan <peng.fan@nxp.com>`

**Comment by jforissier** [comment]
Hi @bryanodonoghue 

Could you please apply MrVan's R-b tag, rebase on top of master and add a commit to update the MAINTAINERS file? (it has a new format now and the new platform should be listed there IMO). Thanks!

**Comment by bryanodonoghue** [comment]
@MrVan I'm going to add us both as maintainers here - provided that's OK with you ?

**Comment by MrVan** [comment]
@bryanodonoghue I am ok.

**Comment by bryanodonoghue** [comment]
I'm going to force update this again.

The failure doesn't appear related to my patch

 **_toolchain-external-custom  Configuring
Cannot execute cross-compiler '/home/travis/optee_repo/build/../toolchains/aarch32/bin/arm-linux-gnueabihf-gcc'
make[2]: *** [/home/travis/optee_repo/out-br/build/toolchain-external-custom/.stamp_configured] Error 1
make[1]: *** [_all] Error 2
make: *** [buildroot] Error 2
make: *** Waiting for unfinished jobs....
The command "(cd ${HOME}/optee_repo/build && $make check CROSS_COMPILE="ccache arm-linux-gnueabihf-" AARCH32_CROSS_COMPILE=arm-linux-gnueabihf- CFG_TEE_CORE_DEBUG=y DUMP_LOGS_ON_ERROR=1)" exited with 2._**

**Comment by jforissier** [comment]
Thanks @bryanodonoghue. The CI failure is indeed unrelated, I'm taking care of it.

**Pull request merged**